rollout.go

```diff
@@ -110,6 +112,27 @@ func (r *client) swapData(data []byte) error {
 	return nil
 }
 
+// RawPercentage returns the raw percentage from the rollout section
+func (r *client) RawPercentage(feature string) (float64, error) {
+	feature = "feature:" + feature
+	r.RLock()
+	value, ok := r.currentData[feature]
+	r.RUnlock()
+
+	if !ok {
+		return 0.0, fmt.Errorf("feature not found: %s", feature)
```

Comment:
Should this `return 0.0, nil` instead of the error? Or maybe it should return a specific typed error defined as a package global so that callers know that they can detect this type of error?

I imagine callers doing things like:
``` golang
percent, err := rolloutClient.RawPercentage("myfeature")
if err == rollout.FeatureNotFoundError {
    percent = defaultPercentage
} else if err != nil {
    return errors.Wrap(err, ...)
}
```
Or this, which would sort of reproduce the behavior in FeatureActive:
``` golang
percent, err := rolloutClient.RawPercentage("myfeature")
if err != nil && err != rollout.FeatureNotFoundError {
    log.Error("Error getting feature percentage: %v", err)
}
```